beacon-chain/sync/validate_beacon_attestation.go

```diff
@@ -68,6 +68,17 @@ func (s *Service) validateCommitteeIndexBeaconAttestation(ctx context.Context, p
 		return pubsub.ValidationIgnore
 	}
 
+	// Verify the block being voted and the processed state is in DB and. The block should have passed validation if it's in the DB.
```

Comment:
please change the validation order in the comment above as well

Reply:
Huh? This comment is correct

Reply:
Huh? This comment is correct

Reply:
this one
https://github.com/prysmaticlabs/prysm/blob/453434240b3b71602a277ab9145abe8d2f5a7c37/beacon-chain/sync/validate_beacon_attestation.go#L24